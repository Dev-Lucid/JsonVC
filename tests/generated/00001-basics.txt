{"title":"","description":"","keywords":"","test":"","author":"","js":"","prepend":[],"append":[],"replace":[]}