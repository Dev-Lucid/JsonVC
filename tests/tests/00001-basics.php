<?
file_put_contents($output_path,json_encode($__jvc['response']));
?>