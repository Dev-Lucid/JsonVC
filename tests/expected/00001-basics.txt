{"title":"","description":"","keywords":"","author":"","js":"","prepend":[],"append":[],"replace":[]}